bin/git-fork

```diff
@@ -55,6 +90,12 @@ if [ "$origin" = true ]; then
 else
     # clone forked repo into current dir
     git clone "${remote_prefix}${user}/${project}.git" "$project"
```

Comment:
@hickey 
Ah ha, this idea is inspired by your `-d $project` check.
If there is `$project` already, we can directly exit the script, skip `git clone`, `cd`, and so on.